Files/Strings/hu-HU/Resources.resw

```diff
@@ -1418,6 +1418,9 @@
   </data>
   <data name="BaseLayoutContextFlyoutSortByOriginalPath.Text" xml:space="preserve">
     <value>Eredeti elérési út</value>
+  </data>
+    <data name="PreemptiveCacheParallelLimit.Header" xml:space="preserve">
```

Comment:
Looks like you edited this file manually. This file is auto generated from `Files.hu-HU.xlf`, so no need to edit it manually. To generate updated file just run project build or just exclude this file from PR and we will do it by ourselves.

Reply:
For more information about translation, visit our [documentation](https://files-community.github.io/docs/#/articles/translating-files).

Reply:
Ok, I restored the file, I don't want to cause any more stupidity, I'd rather leave the update of Files/Strings/hu-HU/Resources.resw to you.